brave/browser/guest_view/tab_view/tab_view_guest.cc

```diff
@@ -154,7 +154,7 @@ void TabViewGuest::Load() {
 void TabViewGuest::NavigateGuest(const std::string& src,
                                  bool force_navigation) {
   // auto tab_helper = extensions::TabHelper::FromWebContents(web_contents());
```

Comment:
can you add a comment for me that the commented code here is for transferrable pinned tabs? Same as the comments in tab_helper?